content/pt-br/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -0,0 +1,263 @@
+---
+title: contêineres de inicialização
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Essa página provê uma visão geral de contêineres de inicialização: especializada nos contêineres que executam antes de contêineres de aplicação em um {{< glossary_tooltip text="Pod" term_id="pod" >}}. Contêineres de inicialização podem conter utilidades ou configurações que não estão presentes em uma imagem de uma aplicação.
+
+Você pode especificar contêineres de inicialização em uma especificação de Pod ao longo de um array de `containers` (que descreve app contêineres).
+
+
+<!-- body -->
+
+## Entendendo contêineres de inicialização
+
+Um {{< glossary_tooltip text="Pod" term_id="pod" >}} pode ter múltiplos contêineres executando aplicações com ele, mas ele também pode ter um ou mais contêineres de inicialização, que são executados antes dos contêineres de aplicação serem iniciados.
+
+Contêineres de inicialização são exatamente como contêineres regulares, exceto:
+
+* Contêineres de inicialização executam sempre até completarem.
+* Cada contêiner de inicialização deve completar com sucesso antes de ir para o próximo início.
+
+Se o contêiner de inicialização do Pod falhar, o kubelet reinicia repetidamente aquele contêiner de inicialização
+
+Contudo, se o Pod tem um `restartPolicy` de Never e o contêiner falha durante a inicialização daquele Pod, Kubernetes trata sobretudo como um Pod que falhou. 
+
+Para especificar um contêiner de inicialização para um Pod, adicione o campo `contêineres de inicialização` dentro do [Pod specification](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec), como um array de items de `contêiner` (similar com o campo de `contêineres`  de aplicação e seu conteúdo).
```

Comment:
Field names should not have been translated in this paragraph. They should be kept as `initContainers`, `container`, and `containers` exactly as they are rendered in the original documentation.